server/src/test/java/io/crate/integrationtests/DDLIntegrationTest.java

```diff
@@ -869,29 +857,21 @@ public void testCreateTableWithGeneratedColumn() throws Exception {
             "create table test (" +
             "   ts timestamp with time zone," +
             "   day as date_trunc('day', ts)) with (number_of_replicas=0)");
-        Map<String, Object> expectedMapping = Map.of(
-            "dynamic", "strict",
-            "_meta", Map.of(
-                "generated_columns", Map.of(
-                    "day", "date_trunc('day', ts)"
-                )
-            ),
-            "properties", Map.of(
-                "ts", Map.of(
-                    "type", "date",
-                    "position", 1,
-                    "oid", 1,
-                    "format", "epoch_millis||strict_date_optional_time"
```

Comment:
I guess this is always the same for timestamp fields, no need to assert, right?

Reply:
yes, we don't expose the information in the same way